server/datastore/mysql/hosts_test.go

```diff
@@ -1583,12 +1583,13 @@ func testHostsAdditional(t *testing.T, ds *Datastore) {
 }
 
 func testHostsByIdentifier(t *testing.T, ds *Datastore) {
+	now := time.Now().UTC().Truncate(time.Second)
```

Comment:
nitpick, don't need to explicitly convert to utc. Will be handled by the sql driver when serializing/deserializing.

Reply:
Ack, copy pasted from other tests :)